test/container_test.py

```diff
@@ -2256,3 +2256,35 @@ def test_set_local_input_concurrency(servicer):
 def test_sandbox_infers_app(servicer, event_loop):
     _run_container(servicer, "test.supports.sandbox", "spawn_sandbox")
     assert servicer.sandbox_app_id == "ap-1"
+
+
+@skip_github_non_linux
+def test_deserialization_error_returns_exception(servicer, client):
+    inputs = [
+        api_pb2.FunctionGetInputsResponse(
+            inputs=[
+                api_pb2.FunctionGetInputsItem(
+                    input_id="in-xyz0",
+                    function_call_id="fc-123",
+                    input=api_pb2.FunctionInput(
+                        args=b"\x80\x04\x95(\x00\x00\x00\x00\x00\x00\x00\x8c\x17",
```

Comment:
instead of a byte string here, can you feed it a pickled object? or is that impossible because it needs to be a different package?

Reply:
yeah would have to somehow pickle the object in another venv than it's being pickled in which sounds annoying but more realistic, but both of these causes serialization errors